src/annotation.js

```diff
@@ -62,7 +75,7 @@ export default {
 	defaults: {
 		drawTime: 'afterDatasetsDraw',
 		dblClickSpeed: 350, // ms
-		annotations: [],
+		annotations: {},
```

Comment:
I think this makes it a breaking change still

Reply:
> 
> 
> I think this makes it a breaking change still

You're right! Sorry

Reply:
> 
> 
> I think this makes it a breaking change still

Do you think it's an issue?

Reply:
It's ok. Someone will file an bug, if it breaks something that we don't want to break. Will do a beta release at some point.